.zenodo.json

```diff
@@ -10,6 +10,10 @@
             "affiliation": "Canadian Food Inspection Agency",
             "name": "Nguyen, Hoang Hai"
         },
+        {
+            "affiliation": "Influenze, Respiratory Viruses, and Coronavirus Section",
+            "name": "Meknas, Abdallah"
```

Comment:
Do you happen to have an ORCID ID? Is this you?

https://orcid.org/0000-0001-9458-7401

Reply:
Yes that's me! I didn't know I had one. Added to the file.